analytic_diffuse/integrators.py

```diff
@@ -0,0 +1,55 @@
+from scipy import integrate
+import numpy as np
+from . import models
+from scipy.special import j0, jn_zeros
+import mpmath as mp
+
+def hankel_solver(model: [callable, str], u: [float, np.ndarray], quad_kwargs={}, high_precision=False, use_points=False, *args, **kwargs):
```

Comment:
Could you add a docstring to explain what this does?

Reply:
Yup, I'll go through and add better docstrings...